lang/cs/Org.Apache.REEF.IO/PartitionedData/FileSystem/FileSystemPartitionConfiguration.cs

```diff
@@ -0,0 +1,43 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System.Collections.Generic;
+using Org.Apache.REEF.IO.FileSystem;
+using Org.Apache.REEF.IO.PartitionedData.FileSystem.Parameters;
+using Org.Apache.REEF.Tang.Formats;
+using Org.Apache.REEF.Tang.Util;
+
+namespace Org.Apache.REEF.IO.PartitionedData.FileSystem
+{
+    public sealed class FileSystemPartitionConfiguration<T> : ConfigurationModuleBuilder
+    {
+        public static readonly RequiredParameter<string> FilePaths = new RequiredParameter<string>();
+
+        public static readonly RequiredParameter<string> FileSystemType = new RequiredParameter<string>();
+
+        public static readonly RequiredImpl<IFileSerializer<T>> FileSerializer = new RequiredImpl<IFileSerializer<T>>();
+
+        public static ConfigurationModule ConfigurationModule = new FileSystemPartitionConfiguration<T>()
```

Comment:
What if IFileSystem or IFileSerializer have extra parameters that need to be binded. In that case we cannot use this module right?